dist/404.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title>Page Not Found</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <style>
+            * {
+                line-height: 1.2;
+                margin: 0;
+            }
+
+            html {
+                color: #888;
+                display: table;
+                font-family: sans-serif;
+                height: 100%;
+                text-align: center;
+                width: 100%;
+            }
+
+            body {
+                display: table-cell;
+                vertical-align: middle;
+                margin: 2em auto;
+            }
+
+            h1 {
+                color: #555;
+                font-size: 2em;
+                font-weight: 400;
+            }
+
+            p {
+                margin: 0 auto;
+                width: 280px;
+            }
+
+            @media only screen and (max-width: 280px) {
+                body,
+                p {
```

Comment:
Not sure if this is an existing standard in use by some preprocessor or linter, but I wonder if there's good reason for not having multiple selectors in the same line? Such as `body, p {`.

Reply:
its a prettier standard. not sure if this one can be configured

Reply:
[Confirmed, it can't](https://github.com/prettier/prettier/issues/3464). And in fact, it [used to be the other way](https://github.com/prettier/prettier/issues/2057).